dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -199,11 +226,14 @@ public UserResource(HttpClient client, SessionFactory sessionFactory, WorkflowRe
     @ApiResponse(responseCode = HttpStatus.SC_NOT_FOUND + "", description = USER_NOT_FOUND_DESCRIPTION)
     @ApiOperation(value = "Get a user by username.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = User.class)
     public User listUser(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
-            @ApiParam("Username of user to return") @PathParam("username") @NotBlank String username) {
+            @ApiParam("Username of user to return") @PathParam("username") @NotBlank String username,
+        @Parameter(name = "include", description = USER_INCLUDE_MESSAGE, in = ParameterIn.QUERY) @ApiParam(value = USER_INCLUDE_MESSAGE) @QueryParam("include") String include) {
```

Comment:
You can specify the allowable query param value(s), like [here](https://github.com/dockstore/dockstore/blob/db15223af216d10633e6fbd08000080d8b60ffde/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java#L281)

Reply:
Is there an option to extend this to multiple values? Let's assume we had multiple fields, say `userProfiles` and `entries` that we'd want to populate. The current implementation would accept `userProfiles, entries` and initialize both fields of the object. 

Using the allowable values schema, it appears that you get confined to a single choice (based on a quick search through hibernate functionality, so I may be missing something), which means in the above scenario you'd need to make two calls, one with `userProfiles` and one with `entries`.

Since we only have `userProfiles`, using allowable values seems more user friendly, but it's a different pattern than the workflow field initializations. No strong feelings either way, but wanted to bring up the other scenarios. Thoughts?

Reply:
Well, you could do the `n!` combinations of possible values as allowable values

Reply:
> Is there an option to extend this to multiple values? 

Whoops, I missed that. My idea doesn't work so well then, other than Denis' suggestion.

On the other hand, this endpoint only currently supports one field, and I'd prefer the generated client not having to know to type "userProfiles", with the correct casing.  But I'm OK if you leave it as is.

Reply:
Not a perfect solution, but the description for the parameter is `Comma-delimited list of fields to include: userProfiles, ...`, which points the user in the right direction. Will leave as-is for now.